Sources/SourceKitLSP/SourceKitLSPServer.swift

```diff
@@ -697,6 +697,8 @@ extension SourceKitLSPServer: MessageHandler {
       await request.reply { try await pollIndex(request.params) }
     case let request as RequestAndReply<BarrierRequest>:
       await request.reply { VoidResponse() }
+    case let request as RequestAndReply<TriggerReIndexRequest>:
```

Comment:
Personally I'd go with `TriggerReindexRequest` as `ReIndex` looks really odd to me. But 🤷

Reply:
I was debating whether to capitalize the `I` and wasn’t sure whether in a sentence re-index or reindex would be the preferred spelling. Changed it to lowercase `i`.